features/cli/add_credential.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Storing AWS access credentials
+
+  In order to manage AWS access credentials
+  As an AWS IAM user
+  I want to store an AWS access credential
+
+  Scenario: Adding a long-term access credential without an MFA device
+    When I run `aws-session add credential` interactively
+    And I type "AKIAXXXXXXXXXXXX"
+    And I type "secretaccesskey1234567890"
+    And I type ""
+    And I type "default"
+    Then the exit status should be 0
+    And the stdout should contain "AWS Secret Access Key"
+    And the stdout should contain "AWS Secret Access Key"
+    And the stdout should contain "MFA device (leave blank for none)"
+    And the stdout should contain "Credential alias"
+
+  Scenario: Adding a temporary security credential without an MFA device
+    When I run `aws-session add credential` interactively
```

Comment:
Also, temp credentials can't require an MFA device (I think) so I guess that component of the scenario name isn't required. If there is a CLI signal that this is temporary the MFA prompt could be swapped for the Session Token prompt.